docs/bir-spec/src/test/resources/test-src/basic/function.bal

```diff
@@ -0,0 +1,3 @@
+function testFunction() {
```

Comment:
We add license header for test cases as well if I am not mistaken

Reply:
ack